anvill/src/Analysis/CrossReferenceResolver.cpp

```diff
@@ -353,12 +353,6 @@ CrossReferenceResolverImpl::ResolveInstruction(llvm::Instruction *inst_val) {
 // Try to resolve a constant to a cross-reference.
 ResolvedCrossReference
 CrossReferenceResolverImpl::ResolveConstant(llvm::Constant *const_val) {
-
-  auto it = xref_cache.find(const_val);
```

Comment:
Why disabling of the fast path of the cache here?

Reply:
The fast path is moved to `ResolveValue` and may not be needed here. But there is no harm having it here as well for cases where `ResolveConstant` is getting called directly.